packages/plugin-ext/src/hosted/node/plugin-host.ts

```diff
@@ -19,6 +19,54 @@ import { RPCProtocolImpl } from '../../api/rpc-protocol';
 import { PluginHostRPC } from './plugin-host-rpc';
 console.log('PLUGIN_HOST(' + process.pid + ') starting instance');
 
+// override exit() function, to do not allow plugin kill this node
```

Comment:
What node? Do you mean that a single plugin should not kill the plugin host process by calling `process.ext`?